gtm/templates/gtm/gtm_head.html

```diff
@@ -0,0 +1,18 @@
+{% if google_tag_id %}
+<!-- Page hiding snippet (recommended) -->
+<style>.async-hide { opacity: 0 !important} </style>
```

Comment:
Should this `async-hide` be here?

Reply:
You can revert it with `git revert f698449ce292171e86b6bd2b0917f340668f2740 ` - this should add a patch / commit that removes / reverses f698449.

Reply:
The push the change to your fork.

Reply:
Ideally the `README.md` would mention the new template tags, but I am not sure if @Lacrymology is fussed about that though.

Reply:
If @olegserov doesn't want to change the docs, I'll be happy to do it after I merge, but since I can't push to his fork, I can't do it in this PR.

Another option is that I merge this into another branch in my repo and fix it myself